pyproject.toml

```diff
@@ -77,6 +77,7 @@ neptune-sklearn = { version = "*", optional = true }
 neptune-tensorflow-keras = { version = "*", optional = true }
 neptune-xgboost = { version = "*", optional = true }
 transformers = { version = "*", optional = true }
+zenml = { version = "*", optional = true, python = "<3.10" }
```

Comment:
Here is inconsistency as we should test on oldest + latest possible (`3.7` + `3.8`) due to what you've configured in integrations workflow.

Reply:
so `zenml = { version = "*", optional = true, python = ">=3.7 <3.9" }` ?